.github/workflows/ci.yml

```diff
@@ -92,31 +92,31 @@ jobs:
           name: packed-tarballs
           path: /home/runner/work/cli/cli/packages/cli/dist
 
-  # sign_deb:
-  #   needs: [pack_deb]
-  #   if: github.ref == 'refs/heads/master' || github.ref_type == 'tag' && startsWith(github.ref_name, 'v' )
-  #   runs-on: ubuntu-latest
-  #   environment: SignDebian
-  #   env:
-  #     HEROKU_DEB_SECRET_KEY: ${{ secrets.HEROKU_DEB_SECRET_KEY }}
-  #     HEROKU_DEB_SIGNING_PASSWORD: ${{ secrets.HEROKU_DEB_SIGNING_PASSWORD }}
-  #     HEROKU_DEB_KEY_ID: ${{ secrets.HEROKU_DEB_KEY_ID }}
-  #     HEROKU_DEB_PUBLIC_KEY: ${{ secrets.HEROKU_DEB_PUBLIC_KEY }}
+  sign_deb:
+    needs: [pack_deb]
+    if: github.ref == 'refs/heads/master' || github.ref_type == 'tag' && startsWith(github.ref_name, 'v' )
+    runs-on: ubuntu-latest
+    environment: SignDebian
+    env:
+      HEROKU_DEB_SECRET_KEY: ${{ secrets.HEROKU_DEB_SECRET_KEY }}
+      HEROKU_DEB_SIGNING_PASSWORD: ${{ secrets.HEROKU_DEB_SIGNING_PASSWORD }}
+      HEROKU_DEB_KEY_ID: ${{ secrets.HEROKU_DEB_KEY_ID }}
+      HEROKU_DEB_PUBLIC_KEY: ${{ secrets.HEROKU_DEB_PUBLIC_KEY }}
+    steps:
+      - uses: actions/checkout@v3
+      - run: sudo mkdir -p /build
+      - uses: actions/download-artifact@v3
+        with:
+          name: packed-deb
+          path: /home/runner/work/cli/cli/packages/cli/dist
+      - run: |
+          cd /home/runner/work/cli/cli/packages/cli/dist/deb
+          /home/runner/work/cli/cli/packages/cli/scripts/sign/deb
+      - uses: actions/upload-artifact@v3
+        with:
+          name: signed-deb
+          path: /home/runner/work/cli/cli/packages/cli/dist
 
-  #   steps:
-  #     - uses: actions/checkout@v3
-  #     - run: sudo mkdir -p /build
-  #     - uses: actions/download-artifact@v3
-  #       with:
-  #         name: packed-deb
-  #         path: /home/runner/work/cli/cli/packages/cli/dist
-  #     - run: |
-  #         cd /home/runner/work/cli/cli/packages/cli/dist/deb
-  #         /home/runner/work/cli/cli/packages/cli/scripts/sign/deb
-  #     - uses: actions/upload-artifact@v3
-  #       with:
-  #         name: signed-deb
-  #         path: /home/runner/work/cli/cli/packages/cli/dist
 
   release-deb-and-tarballs:
     needs: [test, acceptance, pack_tarballs]
```

Comment:
There was a second PR that removed the `sign_deb` job from here that we'll need to put back:
https://github.com/heroku/cli/pull/2123/files